pom.xml

```diff
@@ -606,6 +606,12 @@
             <groupId>com.puppycrawl.tools</groupId>
             <artifactId>checkstyle</artifactId>
             <version>${version.com.puppycrawl.tools.checkstyle}</version>
+            <exclusions>
```

Comment:
So the license checker fails because checkstyle used to refer to the 1.7 tools. jar, which could not be found?